applications/ShapeOptimizationApplication/python_scripts/mesh_controller_with_solver.py

```diff
@@ -65,21 +65,18 @@ def __init__(self, MeshSolverSettings, model):
             },
             "use_automatic_remeshing"     : false,
             "automatic_remeshing_settings": {
-                "step_frequency": 1,
+                "strategy"        : "Optimization",
```

Comment:
```suggestion
                "strategy"        : "optimization",
```

Reply:
Strange enough it does not seem to matter what setting we give here, even with something obviously wrong ("ksjdfh") it gives the same result.

Reply:
Thats because we are forcing the `min` and `max`, and there are no `exception` throws. The `optimization` settings enables MMG `-optim` flag.

Reply:
> Strange enough it does not seem to matter what setting we give here, even with something obviously wrong ("ksjdfh") it gives the same result.

If a not existent  `strategy` is chosen (e.g. a typo) nor remeshing happens...

I have added the wrong reference result, thats why the test succeded, let me update the ref result with the correct settings.